tests/containers/tang/Containerfile

```diff
@@ -1,6 +1,6 @@
 FROM registry.fedoraproject.org/fedora-minimal:41
```

Comment:
would switching to the non-minimal image here be a better move? i.e. would less rpms get downloaded in the `dnf install` below?

Reply:
it does not work